libflux/src/flux/ast/mod.rs

```diff
@@ -440,6 +440,7 @@ pub struct MemberAssgn {
 pub struct StringExpr {
     #[serde(skip_serializing_if = "BaseNode::is_empty")]
     #[serde(default)]
+    #[serde(flatten)]
```

Comment:
Because in Go BaseNode is not a separate child structure like in Rust, it is a base interface. We can only get the same serialized structure by flattening it.